### PR TITLE
Nav updates

### PR DESCRIPTION
**Pull request by tallcoleman**
Update/add nav links for app, landing page, and dashboard:

- [X] Add nav links to dashboard
- [X] Update dashboard nav link on landing page
- [x] Add nav links going to landing page and dashboard to app

### CODE REVIEW

**Comment by tallcoleman** [comment]
@kevinluo201 I added some links to the start page for the app but mostly in a vanilla way - let me know if there's a more preferred way to do things in Gatsby?

File: bikespace_frontend/src/pages/index.tsx [hunk 1]
```diff
@@ -4,50 +4,74 @@ import { button, title, logo, main, mainContent } from "../styles/index.css";
...
+        <nav className={"main-nav"} aria-label="Main">
+          <ul>
+            <li>
+              <a href="https://bikespace.ca/">About BikeSpace</a>
+            </li>
+            <li>
+              <a href="https://dashboard.bikespace.ca/">Explore the Data</a>
+            </li>
+            <li>
+              <a href="https://github.com/bikespace/bikespace/tree/main/bikespace_frontend">
```
Comment (sonalranjit) [inline bikespace_frontend/src/pages/index.tsx:27]: Would it be better to navigate to https://github.com/bikespace/bikespace ?
The frontend folder's README is just the boilerplate gatsby one.

Comment (tallcoleman) [reply bikespace_frontend/src/pages/index.tsx:27]: Should we move that README here then and write a new one for the root folder just outlining the different sub folders? (App, api, dashboard, landing page) happy to draft something for the root if you think this makes sense

Comment (sonalranjit) [reply bikespace_frontend/src/pages/index.tsx:27]: Yeah it's on my list for #78 to cleanup the READMEs. I'll update/add READMEs for the missing ones like frontend, api and landing page.

File: bikespace_frontend/src/pages/index.tsx
Comment (kevinluo201) [inline bikespace_frontend/src/pages/index.tsx:1]: @tallcoleman I think they're fine according to Gatsby's document. (I'm not a Gatsby expert either 😛)https://www.gatsbyjs.com/docs/linking-between-pages/#the-gatsby-link-component

> For external links to pages not handled by your Gatsby site, use the regular HTML <a> tag.

Comment (tallcoleman) [reply bikespace_frontend/src/pages/index.tsx:1]: Thanks Kevin!

**Review by sonalranjit — APPROVED** [review]
looks great

**Pull request merged**